core/src/test/java/com/novoda/noplayer/mediaplayer/AndroidMediaPlayerFacadeTest.java

```diff
@@ -381,13 +385,18 @@ public void givenMediaPlayerIsInPlaybackState_whenGettingPosition_thenReturnsPos
     }
 
     @Test
-    public void givenMediaPlayerIsNotInPlaybackState_whenGettingPosition_thenReturnsZero() {
+    public void givenMediaPlayerIsNotInPlaybackState_whenGettingPosition_thenThrowsIllegalStateException() {
+        thrown.expect(
+                ExceptionMatcher.matches(
+                        "Video must be loaded and not in an error state before trying to interact with the player",
```

Comment:
Can we pull this string out into a field? If it ever changes we'd have to change like three times in the test as it stands